tokio/tests/macros_select.rs

```diff
@@ -481,3 +481,79 @@ async fn mut_on_left_hand_side() {
     .await;
     assert_eq!(v, 2);
 }
+
+#[tokio::test]
+async fn biased_one_not_ready() {
+    let (_tx1, rx1) = oneshot::channel::<i32>();
+    let (tx2, rx2) = oneshot::channel::<i32>();
+    let (tx3, rx3) = oneshot::channel::<i32>();
+
+    tx2.send(2).unwrap();
+    tx3.send(3).unwrap();
+
+    let v = tokio::select! {
+        biased;
+
+        _ = rx1 => unreachable!(),
+        res = rx2 => {
+            assert_ok!(res)
+        },
+        _ = rx3 => {
+            panic!("This branch should never be activated because `rx2` should be polled before `rx3` due to `biased;`.")
+        }
+    };
+
+    assert_eq!(2, v);
+}
+
+#[tokio::test]
+async fn biased_eventually_ready() {
+    use std::future::Future;
+
+    fn ready_on_2nd_poll() -> impl Future<Output = ()> + Unpin {
+        use std::task::Poll;
+
+        let mut polled = false;
+
+        poll_fn(move |ctx| {
+            if polled {
+                Poll::Ready(())
+            } else {
+                polled = true;
+                ctx.waker().wake_by_ref();
+
+                Poll::Pending
+            }
+        })
+    }
+
+    let one = async {};
+    let two = ready_on_2nd_poll();
+    let three = ready_on_2nd_poll();
```

Comment:
This is just a reimplementation of `tokio::task::yield_now`.

Reply:
Aha so it is, I figured there was probably an easier way to do that